Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.yml

```diff
@@ -18,6 +18,25 @@ configuration:
   section: Connect
   required: false
   defaultvalue: 'Cortex XDR - Lite'
+- name: url
+  type: 0
+  display: 'Server URL (copy URL from XDR)'
+  section: Connect
+  required: true
+- displaypassword: API Key ID
+  name: apikey_id_creds
+  type: 9
+  hiddenusername: true
+  display: ''
+  section: Connect
+  required: false
+- display: ''
+  name: apikey_creds
+  type: 9
+  section: Connect
+  required: false
+  displaypassword: API Key
+  hiddenusername: true
```

Comment:
Those weren't part of the integration before?

Reply:
they ere, I wanted to change the order since it wasn't organized well (non mandatory field were at start and inserting API key etc were all the way down- I'll show it on the demo)